src/api/parser/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "parser",
+  "version": "1.0.0",
+  "description": "A service for parsing posts",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
We should add some way to start the server here.  Get rid of `test` and add `start` and maybe `dev` if you want to do anything different there.